### ISSUE TITLE
AppStream description is not interpreted correctly

### ISSUE THREAD

**Issue by glevner** [flathub/flathub#4157]
My app's AppStream description has a paragraph containing an unordered list. On the app's **About** page, that paragraph appears simply as "(null)". The relevant paragraph in the XML file looks like this:
```
    <p>
      <ul>
        <li>
          Use Alt-Enter to enter/exit full screen mode.
        </li>
        <li>
          When playing with a keyboard, the LEFT player uses the E, S, R, D keys
          for buttons, and the F, C, V, B keys for moving; the RIGHT player uses
          the O, K, P, L keys for buttons, and the arrow keys for moving.
        </li>
        <li>
          We strongly recommend you <em>do not</em> play on a keyboard as there
          are "ghosting" issues (some key combinations do not work).
        </li>
      </ul>
    </p>
```
The entire AppStream metainfo file can be found [here](https://github.com/flathub/io.itch.domestique_baston.domestique_baston/blob/master/io.itch.domestique_baston.domestique_baston.metainfo.xml), if it helps.

**Comment by JakobDev** [comment]
This is not valid. A paragraph can't contain a list.

**Comment by glevner** [comment]
If the enclosing `<p>` tag is removed, this should work then?

**Comment by JakobDev** [comment]
Yes, it should work then.

**Comment by glevner** [comment]
> Yes, it should work then.

You're right, thanks. The unordered list works if I remove the `<p>` tag. The `<em>` tag is not interpreted, though; it appears verbatim in the text.

**Comment by razzeee** [comment]
That's an unfortunate side effect of the parser we're using right now, which should change in the coming weeks hopefully

See https://github.com/flathub/website/issues/131

**Comment by hfiguiere** [comment]
The `<em>` tad is now properly handled. So I think this can be closed now.
